index.bs

```diff
@@ -390,6 +390,74 @@ by a custom configuration.
 {}
    ```
 
+# Security Considerations {#security-considerations}
+
+The Sanitizer API is intended to prevent DOM-Based Cross-Site Scripting
+by traversing a DocumentFragment and removing elements and attributes
+according to a configuration. The specified API must not support
+the construction of a Sanitizer object that leaves script-capable markup in
+and doing so would be a bug in the threat model.
+
+That being said, there are security issues which the correct usage of the
+Sanitizer API will not be able to protect against and the scenarios will be
+laid out in the following sections.
+
+## Server-Side Reflected and Stored XSS {#server-side-xss}
+
+<em>This section is not normative.</em>
```

Comment:
editorial: ...  not sure it makes sense to add this to every sub-section. Can we have this once or leave it out?

(I don't know what the actual rules for normatize vs non-normative are, unfortunately. :-) )

Reply:
I've seen specs where the whole _Security Considerations_ block is non-normative and some where all is normative.
I didn't want to be super normative & formal for enumerating some of the attacks, but I also did not want to weak the strong statemement at the beginning with a "non-normative" pretext.

Maybe I could group all the enumerated attacks into a non-normative section like - security considerations
-- general threat model (normative)
-- list of attacks we cant prevent (non-normative).


WDYT?

Reply:
Honestly, I don't know.

The whole normative vs non-normative distinction doesn't seem like a good fit for a security considerations section. The only reason I said something is that I found the repeated "non-normative" sub-headers to be a bit ugly, but that is admittedly not a valid reason. I guess I'd do whatever other specs do; except I'm not sure what that is. :)